src/containers/App/App.js

```diff
@@ -188,6 +189,11 @@ export /* istanbul ignore next */ class App extends Component {
                     component={ImportResources}
                   />
                   <Route path={paths.secrets.all()} exact component={Secrets} />
+                  <Route
```

Comment:
do we need a `byNamespace` version too?

Reply:
yes but I included that part in the other PR